testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -221,6 +221,13 @@ def test_normal(self, vec1, vec2, value):
         assert_equal(mdamath.normal(vec1, vec2), value)
         # add more non-trivial tests
 
+    @pytest.fixture()
+    def test_angle(self, a, b):
```

Comment:
I'd add to the test name to reflect that you're testing a specific edgecase, just to help make things clearer for anyone reading in the future - say something like `test_angle_lower_clip()`